system_tests/bigquery.py

```diff
@@ -29,6 +29,11 @@
 
 DATASET_NAME = 'system_tests' + unique_resource_id()
 
+# We need to wait to stay within the rate limits.
+# The alternative outcome is a 403 Forbidden response from upstream.
+# See: https://cloud.google.com/bigquery/quota-policy
```

Comment:
Note added in 7dfa1ad.

Re:  the error predicate -- I'm reluctant to add it because it makes the test flakier with respect to the back-end, which might change the error text arbitrarily and without notice.